packages/filesystem/src/browser/filesystem-frontend-contribution.ts

```diff
@@ -134,6 +148,22 @@ export class FileSystemFrontendContribution implements FrontendApplicationContri
                 }
             }
         });
+        commands.registerCommand(CommonCommands.NEW_FILE, {
+            execute: async (...args: unknown[]) => {
+                let newFileName = undefined;
+                if (args !== undefined && typeof args[1] === 'string') {
```

Comment:
nit: The order in which we are processing the arguments seems arbitrary. I think resolving all arguments first might be a nice idea, e.g., index 0 before 1 etc. to get more clarity on the method.

Reply:
Sure, I will change the args order anyway.